DataSet strict mode, UMS Namespaces, Policy Filters, Delete project (#28)
jexia/docs: docs/dataset/README.md

- The name of your dataset can contain only Latin characters and digits.
- The name of the dataset has to start with a character.

> **Strict mode** - inform platform to apply strict rule about input data validation. If input object will have some fields which does not exist in  dataset schema insert / update operation will be rejected.   

Reviewer: ```suggestion
**Strict mode** - informs the platform to apply strict rules about input data validation. If an input object has extra fields that do not exist in the dataset schema's insert or update operation, it will be rejected.   
```
